pandas/tests/test_common.py

```diff
@@ -122,11 +131,12 @@ def test_standardize_mapping():
 def test_compression_size(obj, method, compression_only):
 
     with tm.ensure_clean() as filename:
-        getattr(obj, method)(filename, compression=compression_only)
-        compressed = os.path.getsize(filename)
-        getattr(obj, method)(filename, compression=None)
-        uncompressed = os.path.getsize(filename)
-        assert uncompressed > compressed
+        with catch_to_csv_depr():
+            getattr(obj, method)(filename, compression=compression_only)
+            compressed = os.path.getsize(filename)
+            getattr(obj, method)(filename, compression=None)
+            uncompressed = os.path.getsize(filename)
+            assert uncompressed > compressed
```

Comment:
why would those cases raise any warnings? 
(and same for the others below)

Reply:
Same answer as [above](https://github.com/pandas-dev/pandas/pull/21896#discussion_r205529792).